sematic/api/endpoints/tests/test_metrics.py

```diff
@@ -124,8 +125,8 @@ def test_run_state_changed(persisted_run: Run):  # noqa: F811
             {
                 "metric_name": "foo",
                 "series": [
-                    [0, [datetime(2023, 4, 11).timestamp()]],
-                    [1, [datetime(2023, 4, 12).timestamp()]],
+                    [0, [str(datetime(2023, 4, 11).timestamp())]],
```

Comment:
The typing of the labels is listed many places as a string, and seems to always be a string except when timestamps are used (though the type hints never account for the possibility of float/int values). This change and similar ones makes the types consistent.